src/main/resources/META-INF/plugin.xml

```diff
@@ -1,6 +1,6 @@
 <!-- Plugin Configuration File. Read more: https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html -->
 <idea-plugin>
-    <id>org.jetbrains.research.testgenie</id>
+    <id>org.jetbrains.research.testspark</id>
     <name>TestSpark</name>
```

Comment:
Won't it break something on the Plugin Marketplace? What are the consequences of changing the plugin id?

CC: @pderakhshanfar, @arksap2002.

Reply:
<img width="712" alt="Screenshot 2024-06-25 at 2 53 22 AM" src="https://github.com/JetBrains-Research/TestSpark/assets/47223481/13483a9b-ec79-4b5e-998a-5a66746985dd">

I think, we should keep `testgenie`

Reply:
This is our unique ID and should not be changed. There is no way to change it on the marketplace as well. Please keep it as it is

Reply:
Interestingly, renaming this back + adjusting some obvious things in other `*.xml` files lead to the error that we can not find other `*.xml` files rn.....
 I am working on this.

Reply:
fixed